modules/serverless-installing-cli-macosx.adoc

```diff
@@ -0,0 +1,30 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-macosx_{context}"]
+= Installing the `kn` CLI for Mac OS X
```

Comment:
So I see "Mac OS X" here, but "macOS" used elsewhere. I don't see "Mac OS X" used anywhere else in our docs repo, so maybe change these instances to "macOS".